tooling/greybox_fuzzer/src/coverage.rs

```diff
@@ -0,0 +1,751 @@
+                }));
+                total_features += 2;
+            }
+            // Binary operations (we need comparisons)
+            &BrilligOpcode::BinaryFieldOp { destination: _, op, .. } => match op {
+                acvm::acir::brillig::BinaryFieldOp::Add
+                | acvm::acir::brillig::BinaryFieldOp::Sub
+                | acvm::acir::brillig::BinaryFieldOp::Mul
+                | acvm::acir::brillig::BinaryFieldOp::Div
+                | acvm::acir::brillig::BinaryFieldOp::IntegerDiv => {}
+                // Equality and LessThan(Equals) are comparisons that interest us
+                acvm::acir::brillig::BinaryFieldOp::Equals
+                | acvm::acir::brillig::BinaryFieldOp::LessThan
+                | acvm::acir::brillig::BinaryFieldOp::LessThanEquals => {
+                    let features_per_comparison= 1 /*true */+1/*false */+255 /*possible bits() results*/;
+
+                    // Insert features for each potential difference log
+                    for i in 0..features_per_comparison {
+                        feature_to_index_map
+                            .insert((opcode_index, usize::MAX - i), total_features + i);
+                    }
+                    coverage_items.push(BrilligCoverageItemRange::Comparison(CmpCoverageRange {
+                        index: total_features,
+                        bits: 254,
+                    }));
+                    total_features += features_per_comparison;
+                }
+            },
+            // Binary operations (we need comparisons)
+            &BrilligOpcode::BinaryIntOp { destination: _, op, bit_size, .. } => match op {
+                acvm::acir::brillig::BinaryIntOp::Add
+                | acvm::acir::brillig::BinaryIntOp::Sub
+                | acvm::acir::brillig::BinaryIntOp::Mul
+                | acvm::acir::brillig::BinaryIntOp::Div
+                | acvm::acir::brillig::BinaryIntOp::And
+                | acvm::acir::brillig::BinaryIntOp::Or
+                | acvm::acir::brillig::BinaryIntOp::Xor
+                | acvm::acir::brillig::BinaryIntOp::Shl
+                | acvm::acir::brillig::BinaryIntOp::Shr => {}
+                // Equality and LessThan(Equals) are comparisons that interest us
+                acvm::acir::brillig::BinaryIntOp::Equals
+                | acvm::acir::brillig::BinaryIntOp::LessThan
+                | acvm::acir::brillig::BinaryIntOp::LessThanEquals => {
+                    let features_per_comparison = 1 /*true */+1/*false */+1/*when ilog is zero*/+ match bit_size{
+                    acvm::acir::brillig::IntegerBitSize::U1 => 1,
+                    acvm::acir::brillig::IntegerBitSize::U8 => 8,
+                    acvm::acir::brillig::IntegerBitSize::U16 => 16,
+                    acvm::acir::brillig::IntegerBitSize::U32 => 32,
+                    acvm::acir::brillig::IntegerBitSize::U64 => 64,
+                    acvm::acir::brillig::IntegerBitSize::U128 => 128,
+                };
```

Comment:
It seems there's `impl From<IntegerBitSize> for u32`

Reply:
Thanks!